src/frontend/src/components/Demo/index.tsx

```diff
@@ -1,42 +1,36 @@
-import React from "react";
-import { connect } from "react-redux";
-import { RecordData } from "../RecordSearch/Record/types";
-import store, { AppState } from "../../redux/store";
+import React, { useEffect } from "react";
+import { useAppDispatch, useAppSelector } from "../../redux/hooks";
 import { startDemo } from "../../redux/search/actions";
+import useSetupPage from "../../hooks/useSetupPage";
 import DemoInfo from "./DemoInfo";
 import SearchPanel from "../RecordSearch/SearchPanel";
 import Status from "../RecordSearch/Status";
 import Record from "../RecordSearch/Record";
 import Assumptions from "../RecordSearch/Assumptions";
+// import johnCommonRecord from "../../data/demo/johnCommon";
+// import multipleCharges from "../../data/demo/multipleCharges";
+// import blankRecord from "../../data/blankRecord";
 
-interface Props {
-  record?: RecordData;
-  startDemo: Function;
-}
-
-class Demo extends React.Component<Props> {
-  componentDidMount() {
-    document.title = "Demo - RecordSponge";
-    store.dispatch(this.props.startDemo());
-  }
+export default function Demo() {
+  const dispatch = useAppDispatch();
+  let record = useAppSelector((state) => state.search.record);
+  // record = multipleCharges;
+  // record = johnCommonRecord;
+  // record = blankRecord;
```

Comment:
Probably good to clean up

Reply:
Sorry, left that in there since I'm still not using the backend to provide record data during dev. I figure using Demo is a safe place to visualize changes in the browser. Not ideal. Will try to keep that out of the PRs.